### ISSUE TITLE
Error: "Failed to parseField countryCode"

### ISSUE THREAD

**Issue by gyorgyhunor** [viruschidai/validate-vat#43]
I am receiving a strange error today (which yesterday was not present):
It says: "Failed to parseField countryCode."
I am not sure what happened and I don't understand what happened since yesterday. 

![image](https://user-images.githubusercontent.com/64782245/139397787-91891d44-99f8-4bc3-b35a-5b607ac4bd5c.png)

**Comment by pofider** [comment]
The same for me. Adding the sope response if it helps...
```xml
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><soap:Fault><faultcode>soap:Server</faultcode><faultstring>The given SOAPAction urn:ec.europa.eu:taxud:vies:services:checkVat/checkVat does not match an operation.</faultstring></soap:Fault></soap:Body></soap:Envelope>
```

**Comment by pofider** [comment]
This was fixed yesterday in release 0.8.0.
I believe this can be closed now.
